core/src/types.ts

```diff
@@ -156,53 +156,89 @@ export declare interface CollateralStatus {
     balance?: BigNumber;
 }
 
-export declare interface SurplusAuctionBase {
+export declare type CompensationAuctionActiveState =
+    | 'just-started'
+    | 'have-bids'
+    | 'ready-for-collection'
+    | 'requires-restart';
+export declare type CompensationAuctionInactiveState = 'collected';
+
+export declare type CompensationAuctionActionStates = 'loaded' | 'restarting' | 'bidding' | 'collecting';
+
+export declare interface CompensationAuctionBase {
     id: number;
     network: string;
     fetchedAt: Date;
 }
 
-export declare interface SurplusAuctionActive extends SurplusAuctionBase {
+export declare interface CompensationAuctionTransactionFees {
+    restartTransactionFeeEth: BigNumber;
+    allowanceTransactionFeeEth: BigNumber;
+    bidTransactionFeeEth: BigNumber;
+    collectTransactionFeeEth: BigNumber;
+    authTransactionFeeEth: BigNumber;
+    combinedBidFeesEth: BigNumber;
+    allowanceTransactionFeeDai: BigNumber;
+    restartTransactionFeeDai: BigNumber;
+    bidTransactionFeeDai: BigNumber;
+    collectTransactionFeeDai: BigNumber;
+    authTransactionFeeDai: BigNumber;
+    combinedBidFeesDai: BigNumber;
+}
+
+export declare interface SurplusAuctionActive extends CompensationAuctionBase {
```

Comment:
The data inside this and `DebtAuctionActive` are nearly identical. Do we want to try and move this into a combined `CompensationAuctionActive` type or do we want to keep them seperated?

Reply:
I would prefer to keep them split, because even though there're similarities, I don't think we should mix up as much fields as possible:

 - Debt auctions and surplus auctions are similar, but are (acc. to my perseption) different + unrelated entities and hence the data structures should be more-or-less decoupled.
 - Extracting the fields into the common ancestor usually would have to be a solid design decision. Here i do not see enough incentive: We use different contracts to extract the data about the contracts, the varialbes and methods from the contracts are named differently, ... (maybe more depth of difference that i don't remember now).
 Parsing the data that we receive (e.g. to determine state). So there should be two different functions that extract the info from the contracts. It would be odd to have these 2 different functions initialize the objects that are closely related (e.g. have 8/9 similar fields). In other words, unless initialization is very similar, i don't see the great benefit of putting all the common fields we've come up with into the same interface.

Reply:
I agree!